website/frontend/src/components/pages/Message/Message.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { VFC, useEffect, useState } from 'react';
+import { CreateType } from '@gear-js/api';
+import { useParams, useHistory } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Metadata } from '@gear-js/api';
+import clsx from 'clsx';
+import { Loader } from 'react-feather';
+import { formatDate } from 'helpers';
+import { RootState } from 'store/reducers';
+import { getProgramAction, resetProgramAction, getMessageAction, resetMessageAction } from 'store/actions/actions';
+import backIcon from 'assets/images/arrow_back_thick.svg';
+import './Message.scss';
+
+type Params = { id: string };
+
+export const Message: VFC = () => {
```

Comment:
It's  better use FC in this case because you not provide return type [explicitly](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/)